### PR TITLE
feat: Added alert

### PR DESCRIPTION
**Pull request by muskaanshah**
Added alert when user adds an item to the cart / wishlist

### CODE REVIEW

File: src/components/Alert/Alert.jsx [hunk 1]
```diff
@@ -1,26 +1,41 @@
...
 import "./alert.css";
 
-function Alert({ type, msg }) {
+function Alert() {
+	const { alertState, alertDispatch } = useAlert();
 	let alertClass = "";
 	let spanClass = "info";
 
-	if (type === "success") {
+	if (alertState.alertType === "success") {
```
Comment (rutvikpumak) [inline src/components/Alert/Alert.jsx]: instead of doing alertState.alertType everywhere destruct it once used it accordingly

**Review by rutvikpumak — APPROVED** [review]
Suggest just once change and rest of the code looks clean and feat working as expected

File: src/components/Alert/Alert.jsx (hunk 1 above)
Comment (muskaanshah) [reply src/components/Alert/Alert.jsx]: Okay, thank you

**Review by manismk — APPROVED** [review]
code is clean and easy to understand.

But the Preview link is broken

Muskaan's reply: Fixed the issue, thank you

**Pull request merged**